### ISSUE TITLE
Error: You must use a valid syntax option, either: scss, less or sugarss

### ISSUE THREAD

**Issue by Eschon** [adrianhall/stylelint-loader#15]
When I try to use stylelint-loader I get the following error on every css file:

```
Module build failed: Error: You must use a valid syntax option, either: scss, less or sugarss
```

Here's my webpack config:

```
module: {
  preloaders: [
    {
      test: /\.css$/,
      loader: 'stylelint',
      include: srcPath,
    },
  ],
  loaders: [
    {
      test: /\.css$/,
      include: srcPath,
      loader: 'style!css!postcss'
    },

    ...
  ],
},
stylelint: {
  configFile: path.join(__dirname, relativePath, '.stylelintrc'),
},
```

My .stylelintrc: 

```
{
  "extends": "stylelint-config-standard"
}
```

Using the CLI everything works fine.

Here some information about the package versions I have:

node: 6.3.1
stylelint: 7.1.0,
stylelint-loader: 6.1.0,
webpack: 1.13.1

Please let me know if you need any additional information.

**Comment by adrianhall** [comment]
Do you have an example file that is failing?  Also, does it fail when running it through stylelint standalone?

**Comment by Eschon** [comment]
It happens with every css file in my project. I suspect that it'll happen for any other css file that I ad. Sadly I can't test it for a while because I'm on vacation right now.

**Comment by adrianhall** [comment]
Ok - I'll spin something up.  My simple tests didn't catch this, so I need to dig deeper.

**Comment by thibthib** [comment]
As I understand it, you don't have to pass the `syntax` option anymore (here : https://github.com/adrianhall/stylelint-loader/blob/master/index.js#L89) as the syntax will be automatically inferred from the file extension.

In the [stylelint 7.0.0 changelog](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#700) :

```
Changed: non-standard syntaxes are automatically inferred from file extensions .scss, .less, and .sss. 
```

Am I wrong ? More info here : https://github.com/stylelint/stylelint/issues/1676

**Comment by adrianhall** [comment]
Good point. 

I'm closing down stylelint-loader in preference of the postcss-loader and/or the plugin.
